gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -444,6 +469,33 @@ gst_tensor_converter_set_property (GObject * object, guint prop_id,
       self->silent = g_value_get_boolean (value);
       silent_debug ("Set silent = %d", self->silent);
       break;
+    case PROP_MODE:
+    {
+      const gchar *param = g_value_get_string (value);
+      const converter_custom_cb_s *ptr = NULL;
+      gchar **strv = g_strsplit_set (param, ":", -1);
+
+      if (strv[0] == NULL || strv[1] == NULL) {
+        nns_logw ("Tensor converter mode option is incorrect."
+            "Please specify mode option as <MODE>:<MODE_OPTION>");
+        g_strfreev (strv);
+        break;
+      }
+
+      self->mode = g_strdup (strv[0]);
```

Comment:
For future work, how about adding new enum for mode (enum _NONE=0, _CUSTOM=1, todo add another mode later) to avoid string cmp?

Reply:
Changed,